apps/web/src/containers/LandingPageView/components/our-backers/components/image-slider.tsx

```diff
@@ -0,0 +1,131 @@
+import 'slick-carousel/slick/slick-theme.css';
+import { Box, styled } from '@mui/material';
+import panteraSvg from '@notional-finance/assets/marketing/partners/pantera-new.svg';
+import spartanSvg from '@notional-finance/assets/marketing/partners/spartan-new.svg';
+import ideoCoLabSvg from '@notional-finance/assets/marketing/partners/ideo-colab-new.svg';
+import oneCSvg from '@notional-finance/assets/marketing/partners/1c-new.svg';
+import coinbaseVenturesSvg from '@notional-finance/assets/marketing/partners/coinbase-ventures-new.svg';
+import parafiSvg from '@notional-finance/assets/marketing/partners/parafi-new.svg';
+import nascentSvg from '@notional-finance/assets/marketing/partners/nascent-new.svg';
+
+export const ImageSlider = () => {
+  const settings = {
+    dots: false,
+    infinite: true,
+    speed: 5000,
+    slidesToShow: 4,
+    slidesToScroll: 1,
+    autoplay: true,
+    autoplaySpeed: 0,
+    cssEase: 'linear',
+    arrows: false,
+    responsive: [
+      {
+        breakpoint: 1200,
+        settings: {
+          slidesToShow: 3,
+          slidesToScroll: 1,
+        },
+      },
+      {
+        breakpoint: 1000,
+        settings: {
+          slidesToShow: 2,
+          slidesToScroll: 1,
+        },
+      },
+      {
+        breakpoint: 500,
+        settings: {
+          slidesToShow: 1,
+          slidesToScroll: 1,
+        },
+      },
+    ],
+  };
+
+  return (
+    <SliderContainer>
+      <SliderShadows
+        sx={{
+          background:
+            'linear-gradient(90deg, #012E3A 47.07%, rgba(1, 46, 58, 0) 100%)',
+          marginRight: '-100px',
+        }}
+      ></SliderShadows>
+      <StyledSlider {...settings}>
+        <SliderImgContainer>
+          <img src={panteraSvg} alt="slide1" />
+        </SliderImgContainer>
+
+        <SliderImgContainer>
+          <img src={spartanSvg} alt="slide2" />
+        </SliderImgContainer>
+
+        <SliderImgContainer>
+          <img src={ideoCoLabSvg} alt="slide3" />
+        </SliderImgContainer>
+
+        <SliderImgContainer>
+          <img src={oneCSvg} alt="slide4" />
+        </SliderImgContainer>
+
+        <SliderImgContainer>
+          <img src={coinbaseVenturesSvg} alt="slide5" />
+        </SliderImgContainer>
+
+        <SliderImgContainer>
+          <img src={parafiSvg} alt="slide6" />
+        </SliderImgContainer>
+
+        <SliderImgContainer>
+          <img src={nascentSvg} alt="slide7" />
+        </SliderImgContainer>
+      </StyledSlider>
+      <SliderShadows
+        sx={{
+          background:
+            'linear-gradient(270deg, #012E3A 47.07%, rgba(1, 46, 58, 0) 100%)',
+          marginLeft: '-100px',
+        }}
+      />
+    </SliderContainer>
+  );
+};
+
+const SliderContainer = styled(Box)(
+  ({ theme }) => `
+  width: ${theme.spacing(160)};
+  display: flex;
+  margin: auto;
+  justify-content: center;
+  padding-bottom: ${theme.spacing(15)};
+
+  ${theme.breakpoints.down(1280)} {
```

Comment:
Should this use a theme based breakpoint rather than a hardcoded px value?

Reply:
This must be a hardcoded value to be able to align properly. This is because of the faded overlay on either end forcing this component to need to be longer to line up with the rest of the page.